lib/compilers/_all.ts

```diff
@@ -63,11 +63,12 @@ export {
     DotNetNativeAotCompiler,
 } from './dotnet.js';
 export {EDGCompiler} from './edg.js';
-export {EllccCompiler} from './ellcc.js';
+export {EWARMCompiler} from './ewarm.js';
+export {ElixirAsmCompiler} from './elixirasm.js';
 export {ElixirCompiler} from './elixir.js';
-export {ErlangCompiler} from './erlang.js';
+export {EllccCompiler} from './ellcc.js';
 export {ErlangAsmCompiler} from './erlangasm.js';
-export {EWARMCompiler} from './ewarm.js';
+export {ErlangCompiler} from './erlang.js';
```

Comment:
We like to keep this alphabetically sorted

Reply:
Sorry, I did a sort line but it was case sensitive

Reply:
Done!